roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -120,6 +226,33 @@ resources:
       router_id: { get_resource: router }
       subnet_id: { get_resource: subnet }
 
+{% if openshift_use_kuryr|default(false)|bool %}
+  pod_subnet_interface:
+    type: OS::Neutron::RouterInterface
+    properties:
+      router_id: { get_resource: router }
+      subnet_id: { get_resource: pod_subnet }
+
+  service_router_port:
+      type: OS::Neutron::Port
+      properties:
+        network: { get_resource: service_net}
+        fixed_ips:
+          - subnet: { get_resource: service_subnet }
+            ip_address: {{ openshift_openstack_kuryr_service_subnet_cidr | ipaddr('-2') }}
```

Comment:
I guess similarly here, this probably needs to be an address, not a CIDR so we should tack ` | ipaddr('address')` at the end.

Reply:
That's right. I just tested this filter locally.

Reply:
Done